boards/imix/src/main.rs

```diff
@@ -5,23 +5,9 @@
 
 #![no_std]
 #![no_main]
-#![feature(in_band_lifetimes)]
+#![feature(in_band_lifetimes, uniform_paths)]
```

Comment:
`in_band_lifetimes` is stablized in `rust 1.31` consider to bumping the toolchain

https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1310-2018-12-06

Reply:
We're on 1.32, and it doesn't compile if I remove the `in_band_lifetimes` feature.

Reply:
nvm, I was wrong [in_band_lifetimes](https://github.com/rust-lang/rust/issues/44524) is still not stabilized but [impl_header_lifetime_elision](https://github.com/rust-lang/rust/issues/15872) is.